core/cap-0047.md

```diff
@@ -57,11 +57,26 @@ enum ContractMetadataType
     METADATA_TYPE_V0 = 0
 };
 
+union SCTypes switch (SCValType type)
```

Comment:
This probably shouldn't be a plural, since it is a union, it represents one of, not many of.
```suggestion
union SCType switch (SCValType type)
```